samples/client/petstore/rust/reqwest/petstore-async/.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-6.1.0-SNAPSHOT
\ No newline at end of file
+6.0.0-SNAPSHOT
```

Comment:
looks like you didn't create this change based on the latest master.

can you please pull the latest master into your branch and update the samples again? thanks.